### PR TITLE
VOLDEV-110 - Fix avatars in related threads module

### PR DESCRIPTION
**Pull request by onei**
@Grunny Note: this is untested due to avatar bugs on the voldev devbox.

### CODE REVIEW

**Comment by Grunny** [comment]
@RafLeszczynski You assigned this to yourself as while ago, were you going to review it? :)

**Comment by Grunny** [comment]
@garthwebb I think the current Forums are still owned by Social, can you have a look? :)

**Comment by mszabo-wikia** [comment]
@macbre @mixth-sense Forums is SUS now! 😄 

https://wikia-inc.atlassian.net/browse/VOLDEV-110

File: extensions/wikia/Forum/templates/Forum_forumRelatedThreads.php [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
-			<?php foreach ( $messages as $message ): ?>
+			<?php foreach ( $messages as $message ) { ?>
 			<li>
-				<?= AvatarService::renderAvatar( $message['message']->getUser()->getName(), 20 ); ?>
+				<?php
+				$messageLast = empty( $message['reply'] ) ? $message['message'] : $message['reply'];
+				$user = Html::element(
+					'a',
+					[
+						'href' => $messageLast->getUser()->getUserPage()->getFullUrl()
```
Comment (macbre) [inline extensions/wikia/Forum/templates/Forum_forumRelatedThreads.php:12]: Do we need a full URL here (with domain)? If not, let's use `getLocalURL`.

**Review by macbre — APPROVED** [review]

**Comment by mszabo-wikia** [comment]
https://wikia-inc.atlassian.net/browse/SUS-1533 has been set up to manage QA.

**Pull request merged**